internal/wasm/module.go

```diff
@@ -559,9 +559,15 @@ func paramNames(localNames IndirectNameMap, funcIdx uint32, paramLen int) []stri
 func (m *Module) buildMemory() (mem *MemoryInstance) {
 	memSec := m.MemorySection
 	if memSec != nil {
+		min := MemoryPagesToBytesNum(memSec.Min)
+		capacity := MemoryPagesToBytesNum(memSec.Cap)
+		if capacity == 0 && min != 0 {
```

Comment:
this is a little bomb, I'll remove it. Basically spectests aren't going through normal code paths, but it is better to fix them vs leave this around as it confuses that we've already validated.